lib/highline_wrapper/open_ended_question.rb

```diff
@@ -7,7 +7,7 @@ class OpenEndedQuestion < Question
     class << self
       def ask(prompt, options)
         answer = ask_highline(prompt, secret: options[:secret]).to_s
-        puts unless answer.empty? && options[:secret]
+        puts if options[:include_newline]
```

Comment:
Just don't worry about the fact that the secret will automatically add a newline. I made a note in the README and don't really care so much about this weird edge-case.